imap_processing/spice/repoint.py

```diff
@@ -26,18 +26,22 @@ def get_repoint_data() -> pd.DataFrame:
 
     Returns
     -------
-    repoint_df : pd.DataFrame
+    repoint_df : pandas.DataFrame
         The repointing csv loaded into a pandas dataframe. The dataframe will
         contain the following columns:
-            - `repoint_start_sec`: Starting MET seconds of repoint maneuver.
-            - `repoint_start_subsec`: Starting MET milliseconds of repoint maneuver.
-            - `repoint_start_met`: Floating point MET of repoint maneuver start time.
-            Derived by combining `repoint_start_sec` and `repoint_start_subsec`.
-            - `repoint_end_sec`: Ending MET seconds of repoint maneuver.
-            - `repoint_end_subsec`: Ending MET milliseconds of repoint maneuver.
-            - `repoint_end_met`: Floating point MET of repoint maneuver end time.
-            Derived by combining `repoint_end_sec` and `repoint_end_subsec`.
-            - `repoint_id`: Unique ID number of each repoint maneuver.
+
+            * `repoint_start_sec_sclk`: Starting MET seconds of repoint maneuver.
+            * `repoint_start_subsec_sclk`: Starting MET microseconds of repoint
+              maneuver.
+            * `repoint_start_met`: Floating point MET of repoint maneuver start time.
+              Derived from `repoint_start_sec_sclk` and `repoint_start_subsec_sclk`.
+            * `repoint_start_time_utc`: UTC time of repoint maneuver start time.
```

Comment:
Looks like we just need update in the doc string.
```suggestion
            * `repoint_start_utc`: UTC time of repoint maneuver start time.
```